Source/DafnyLanguageServer/Language/DafnyProgramVerifier.cs

```diff
@@ -66,7 +66,10 @@ public class DafnyProgramVerifier : IProgramVerifier {
           cancellationToken.ThrowIfCancellationRequested();
           VerifyWithBoogie(boogieProgram, cancellationToken);
         }
-        return printer.SerializedCounterExamples;
+        return new VerificationResult(
+          Verified: printer.SerializedCounterExamples == null,
```

Comment:
Shouldn't this be `Verified: true` ?

Reply:
No, since the property should only be `true` if the verification succeeded.

Reply:
I see. Have you considered having a computed property `Verified => SerializedCounterExamples == null` instead of a separate field `Verified` ?

Reply:
Good input. Since it's now the responsibility of `TextDocumentLoader` to decide if a verification should be run or not, I've adapted the code to use the property-expression you suggested.